core/src/main/scala/org/apache/spark/scheduler/SchedulableBuilder.scala

```diff
@@ -61,6 +61,7 @@ private[spark] class FairSchedulableBuilder(val rootPool: Pool, sc: SparkContext
 
   val schedulerAllocFile = sc.conf.get(SCHEDULER_ALLOCATION_FILE)
   val DEFAULT_SCHEDULER_FILE = "fairscheduler.xml"
+  val DEFAULT_SCHEDULER_TEMPLATE_FILE = "fairscheduler-default.xml.template"
```

Comment:
To avoid any conflicts in the existing production jobs, this PR provide and use new file as `.xml.template`.